bit_manipulation/add_largest_power_of_2_less_than_or_equal_to_a_given_number.py

```diff
@@ -0,0 +1,45 @@
+"""
+Fixes #9374
+Name : Sharnabh Banerjee
+
+"""
+
+
+def add_largest(num: int) -> int:
```

Comment:
Can you rename the file to be less wordy.
This function doesn't accomplish what the issue is referencing.